config/initializers/file_set_derivatives_overrides.rb

```diff
@@ -37,5 +37,16 @@ def create_image_derivatives(filename)
                                                             url: derivative_url('thumbnail'),
                                                             layer: 0 }])
   end
-  
-end
\ No newline at end of file
+
+  def create_video_derivatives(filename)
+    original_size = "#{file_set.width.first}x#{file_set.height.first}"
```

Comment:
Great point!  Hydra Derivatives has a default of [320x240](https://github.com/samvera/hydra-derivatives/blob/main/lib/hydra/derivatives/processors/video/config.rb#L59).  Maybe use that as a fallback if values are `nil` or `[]`?

Reply:
@jeremyf how about...
```rb
def create_video_derivatives(filename)
    DEFAULT_VIDEO_SIZE = '320x240'
    width = Array(file_set.width).first
    height = Array(file_set.width).first
    original_size = "#{width}x#{height}"
    size = width.nil? && height.nil? ? : DEFAULT_SIZE : original_size
    Hydra::Derivatives::Processors::Video::Processor.config.size_attributes = size
  # ...
  # ...
  # ...
end
```

Reply:
You'll want to move that constant outside of the method definition but otherwise this ensures that you must have a calculated height and width.  If not, you fall back to the default video size.

Also add a comment on the constant that has a `@see https://github.com/samvera/hydra-derivatives/blob/main/lib/hydra/derivatives/processors/video/config.rb#L59` that way folks can connect the dots on why this magic number.